### PR TITLE
Update VS Code / DevTools section with latest info/screenshots

### PR DESCRIPTION
**Pull request by DanTup**
This PR updates the text and screenshots for running DevTools in VS Code to reflect the current stable behaviour. The existing text/screenshots are from before DevTools was embedded, and before it was shipped inside the SDK.

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/development/tools/devtools/vscode.md
Comment (johnpryan) [inline src/development/tools/devtools/vscode.md:33]: It would be great to link to [the recommended settings](https://dartcode.org/docs/recommended-settings/) for context here.

**Review by johnpryan — APPROVED** [review]

**Review by kenzieschmoll — APPROVED** [review]

File: src/development/tools/devtools/vscode.md
Comment (DanTup) [reply src/development/tools/devtools/vscode.md:33]: I can add this, although I'm not sure how to frame it because it's not really related do this - none of those settings are in the recommended settings (because the recommended settings are the defaults). The recommend settings are really things that aren't Dart-specific and/or might be surprising if we configured them by default.

We could link to the [full settings list](https://dartcode.org/docs/settings/), something like:

> A full list of Dart/Flutter settings are available [here](https://dartcode.org/docs/settings/), along with some recommendation settings for Dart/Flutter in VS Code [here](https://dartcode.org/docs/recommended-settings/).

Although I don't know if for the general settings it'd be better to direct users to just browse them in VS Code (the settings editor has some groupings/filters and a search box).

**Comment by DanTup** [comment]
@johnpryan I'm not sure what the process is for merging here - should I do it, or do you want to? If me, let me know if you want me to change anything based on the comments above before I do. Thanks!

File: src/development/tools/devtools/vscode.md
Comment (johnpryan) [reply src/development/tools/devtools/vscode.md:33]: When I read this, I wasn't sure what a setting was, so I googled "Dart VSCode settings" or something. Any of those options sounds good to me.

**Comment by johnpryan** [comment]
I just have one comment. We also need a review from a technical writer cc: @sfshaza2

**Comment by DanTup** [comment]
Gotcha - I'll wait for instructions that it's good to merge (or one of you are free to merge when happy with it) then. Thanks! :-)

**Review by sfshaza2 — APPROVED** [review]
lgtm. Thanks, @DanTup!

**Pull request merged**
